@kiva/kv-shop/src/basketCredits.ts

```diff
@@ -74,7 +82,7 @@ export async function applyPromoCredit(
 		variables: { ...options?.variables },
 	});
 
-	return !!data?.shop?.addCreditByType;
+	return Promise.resolve(data);
```

Comment:
I don't know that we need to wrap this one b/c `data` is the result of the query promise...

Reply:
done